config.yaml

```diff
@@ -1,50 +1,46 @@
 options:
   site:
     type: string
-    description: >-
-      The site name, e.g. "mysite.local"
-
-      This setting is required.
+    description: >
```

Comment:
What about using the `|` character? We tend to do that in most of our yaml files

Reply:
@jdkandersson , wouldn't it be better to keep use `>-`? See https://github.com/canonical/content-cache-k8s-operator/pull/4#discussion_r1031175775

Reply:
I found some issues related to that (https://github.com/canonical/charmhub.io/search?q=%22config.yaml%22&type=issues) but not a guideline so I believe that it's up to us to decide.

">-" remove the new line in the end but it seems that doesn't make much difference on Charmhub (or does?). I would suggest keeping ">" for one line and "|" for multi-lines. And maybe add some note in the Contributing guide about it.
What do you think?

Reply:
I would stick to yaml standards if possible

Reply:
@amandahla if you could write up some guidance in the contributing doc that would be great! I wasn't aware there were intricacies around how charmhub will interpret this